### PR TITLE
Add William Barbosa as an author

### PR DESCRIPTION
**Pull request by willsb**
:tada: 
By sending this pull request to add my blog I verify that I adhere to the Planet Xamarin blog guidelines:

- [X] I have a valid blog & RSS URL, both using HTTPS with a valid certificate
- [X] I host NO malicious or offensive content on the blog (including photos, swearing, etc.)
- [X] My blog is active with at least 3 Xamarin related blog posts in the last 6 months*
- [X] In addition to the previous guideline; it should be apperant that the blog is active for a longer period of time and the posts on it are not fabricated in a small amount of time to comply to the '3 posts in last 6 months' guideline.
- [X] I have applied a filter (Optional: if blog is only about Xamarin)
- [X] I understand, if I delete my blog, I will be deleted from Planet Xamarin
- [X] I understand, my blog may be removed at any time if any of these guidelines are broken.
- [X] The submitted blog is not owned by a company which sole purpose it is to advertise a product of productline.

*I recently migrated my blog from github pages to medium, hence why there aren't many older blog posts there. I, however, am very much active 😄

### CODE REVIEW

**Comment by Cheesebaron** [comment]
Fill out the issue template please :)

**Comment by willsb** [comment]
Done 👍

File: src/Firehose.Web/Authors/WilliamBarbosa.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    public class WilliamBarbosa : IAmAMicrosoftMVP
+    {
+        public string FirstName => "William";
+        public string LastName => "Barbosa";
+        public string StateOrRegion => "Santos, Brasil";
+        public string TwitterHandle => "willdotnet";
+        public string GitHubHandle => "willsb";
+
+        public string ShortBioOrTagLine
+            => "Mobile Developer, Microsoft MVP, Blogger, Speaker, Xamarin Enthusiast, Co-Founder/Host for the Monkey Nights Podcast/Webcast, MvvmCross contributor and usually the coolest hair in the room";
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/WilliamBarbosa.cs]: Can you please shorten the bio, it breaks the layout.

Comment (willsb) [reply src/Firehose.Web/Authors/WilliamBarbosa.cs]: Done

**Pull request merged**